content/en/observability_pipelines/setup/datadog_with_archiving.md

```diff
@@ -0,0 +1,529 @@
+  
+[1]: https://hub.docker.com/r/datadog/observability-pipelines-worker
+[2]: /resources/yaml/observability_pipelines/archives/pipeline.yaml
+{{% /tab %}}
+{{% tab "AWS EKS" %}}
+1. Download the [Helm chart][1] for AWS EKS.
+
+2. In the Helm chart, replace the `datadog.apiKey` and `datadog.pipelineId` values to match your pipeline and use {{< region-param key="dd_site" code="true" >}} for the `site` value. Replace `${DD_ARCHIVES_SERVICE_ACCOUNT}` in the Helm config with the service account you created earlier. Then, install it in your cluster with the following commands:
+
+    ```shell
+    helm repo add datadog https://helm.datadoghq.com
+    ```
+    ```shell
+    helm repo update
+    ```
+    ```shell
+    helm upgrade --install \
+        opw datadog/observability-pipelines-worker \
+        -f aws_eks.yaml
+    ```
+
+[1]: /resources/yaml/observability_pipelines/archives/aws_eks.yaml
+{{% /tab %}}
+
+{{% tab "APT-based Linux" %}}
+1. Run the following commands to set up APT to download through HTTPS:
+
+    ```
+    sudo apt-get update
+    sudo apt-get install apt-transport-https curl gnupg
+    ```
+
+2. Run the following commands to set up the Datadog `deb` repo on your system and create a Datadog archive keyring:
+
+    ```
+    sudo sh -c "echo 'deb [signed-by=/usr/share/keyrings/datadog-archive-keyring.gpg] https://apt.datadoghq.com/ stable observability-pipelines-worker-1' > /etc/apt/sources.list.d/datadog-observability-pipelines-worker.list"
+    sudo touch /usr/share/keyrings/datadog-archive-keyring.gpg
+    sudo chmod a+r /usr/share/keyrings/datadog-archive-keyring.gpg
+    curl https://keys.datadoghq.com/DATADOG_APT_KEY_CURRENT.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+    curl https://keys.datadoghq.com/DATADOG_APT_KEY_F14F620E.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+    curl https://keys.datadoghq.com/DATADOG_APT_KEY_C0962C7D.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+    ```
+
+3. Run the following commands to update your local `apt` repo and install the Worker:
+
+    ```
+    sudo apt-get update
+    sudo apt-get install observability-pipelines-worker datadog-signing-keys
+    ```
+
+4. Add your keys and the site ({{< region-param key="dd_site" code="true" >}}) to the Worker's environment variables:
+
+    ```
+    sudo cat <<-EOF > /etc/default/observability-pipelines-worker
+    DD_API_KEY=<API_KEY>
+    DD_OP_PIPELINE_ID=<PIPELINE_ID>
+    DD_SITE=<SITE>
+    AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID>
+    AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY>
+    EOF
+    ```
+
+5. Download the [sample configuration file][1] to `/etc/observability-pipelines-worker/pipeline.yaml` on the host.
+
```

Comment:
Should we mention here to update the sample config with the archives bucket and region information? https://github.com/DataDog/documentation/pull/20541/files#diff-7c35f7e1358cf5f87d9f00b6526f58ea5959c488b719387b280f894d5

Reply:
The command on 242 to 248 should be updated, actually ([example](https://a.cl.ly/NQu5mxO9))

So I'd add right under line 247
```
DD_ARCHIVES_BUCKET=<AWS_BUCKET_NAME> 
DD_ARCHIVES_SERVICE_ACCOUNT=<BUCKET_AWS_REGION>
```